include/seqan3/alignment/pairwise/detail/policy_alignment_result_builder.hpp

```diff
@@ -73,27 +75,33 @@ class policy_alignment_result_builder
      * \tparam matrix_coordinate_t The type of the matrix coordinate.
      * \tparam alignment_matrix_t The type of the alignment matrix.
      * \tparam callback_t The type of the callback to invoke.
+     * \tparam transfer_log_fn_t The type of the function which transfers the logged alignment matrix.
      *
      * \param[in] sequence_pair The indexed sequence pair.
      * \param[in] id The associated id.
      * \param[in] score The best alignment score.
      * \param[in] end_positions The matrix coordinate of the best alignment score.
      * \param[in] alignment_matrix The alignment matrix to obtain the trace back from.
      * \param[in] callback The callback to invoke with the generated result.
+     * \param[in] transfer_log The invocable to transfer the logged alignment matrix.
      *
      * \details
      *
      * Generates a seqan3::alignment_result object with the results computed during the alignment. Depending on the
      * \ref seqan3_align_cfg_output_configurations "seqan3::align_cfg::output_*" configuration only the requested values
      * are stored. In some cases some additional work is done to generate the requested result. For example computing
      * the associated alignment from the traceback matrix.
+     *
+     * The first parameter is the logger and is only available if the alignment was run in debug mode. It stores the
```

Comment:
The first?

Reply:
oh, old comment. Because of some rebasing. Thanks